apps/data-scanner/lib/db/migrations/sqls/20220410070449-add-table-sessions-up.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE users
+RENAME COLUMN user_id TO id;
+
+ALTER TABLE tracked_phones
+ADD COLUMN id SERIAL PRIMARY KEY,
+ADD COLUMN session_id NUMERIC,
+ADD UNIQUE (user_id, tracked_phone);
+
+DROP TABLE IF EXISTS sessions;
+CREATE TABLE sessions(
+	id SERIAL PRIMARY KEY,
+    phone VARCHAR(20) NOT NULL,
+    auth_key BYTEA NOT NULL,
+    dc_id TEXT NOT NULL,
+    server_address INET NOT NULL,
```

Comment:
INET support IPv4/6 adrdess types? does server_address could be a URI type?

Reply:
Yes this type resolve exactly this issue.
https://www.postgresql.org/docs/current/datatype-net-types.html